tests/Responses/Expected/checkout/reselect_shipping_response.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": @integer@,
+    "items": @array@,
```

Comment:
Thats true, but also there is no point in mocking it if we are sure about them. Anyway, it is up to you, so if you prefer to leave them this way, I'm ok with it.